src/taipy/config/checker/_checkers/_gLobal_config_checker.py

```diff
@@ -18,17 +18,30 @@
 
 
 class _GlobalConfigChecker(_ConfigChecker):
+    
+    _ACCEPTED_REPOSITORY_TYPES = ["filesystem", "sql"]
```

Comment:
@jrobinAV @bobbyshermi 
I'm thinking that why don't we keep this list empty? Then on taipy-core, we will propagate the list by injecting available repo options. This way, it's easier to expand the supported repo type (different packages provide different supported repo types) while still helps keep the maintenance of possible repo types within their respective package.

Reply:
I believe we can manage this after the merge in another PR. It concernes another ticket related to an enterprise package.
What do you thing @toan-quach ?

Reply:
And so we can implement it ourselves.

Reply:
sure that works for me too :D